vsphere/meta.yaml

```diff
@@ -0,0 +1,10 @@
+browse_categories:
+- Cloud infrastructure and services
+data_signature: vcenter:"*"
```

Comment:
IIRC we typically refrain from having wildcards here. @mstumpfx ?

Reply:
Ok, fixed.